src/app/features/workflow/workflow.component.ts

```diff
@@ -61,4 +61,12 @@ export class WorkflowComponent implements OnInit {
     this.projet = projetOutput;
   }
 
+  validerAction(){
```

Comment:
appel le creerUneTache() c est pas assez explicite.
 Je ne comprend pas ta boucle for ? a quoi sert ton idWorkflow qui = 1 toujours si le numOrdre = 1 ? et sinon ?

Reply:
pour récupérer l'étape workflow la plus à gauche depuis le front